tests/integration/routes/test_view_submitted_response.py

```diff
@@ -58,6 +58,9 @@ def test_enabled(self):
         self.assertInBody("NP10 8XG")
         self.assertIsNotNone(self.get_print_button())
         self.assertIsNotNone(self.get_download_button())
+        self.assertInBody("<th>Question</th>")
+        self.assertInBody("<th>Answer given</th>")
+        self.assertNotInBody("<th>Change answer</th>")
```

Comment:
How come these moved?. These shouldn't be in this test case.

Reply:
The test was failing where it was originally because the timeout was being expired so the table wasn't displayed so I moved it into this test because I thought it was a good place to put it as it was testing all other parts of the page

Reply:
Alright, happy for it to stay.